### PR TITLE
Support BSPWM

### PR DESCRIPTION
**Pull request by joseildofilho**
I Just wrote some code to add BSPWM at the suported envs.

### CODE REVIEW

**Comment by vineetred** [comment]
@joseildofilho don't you need feh installed for this? I would prefer if it didn't depend on feh.

File: src/wallpapers/linux.rs [hunk 1]
```diff
@@ -108,6 +110,12 @@ impl Desktop for DesktopEnvt {
                     ])
                     .output()?;
             }
+
+            DesktopEnvt::BSPWM => {
+                Command::new("feh")
+                    .args(&["--bg-fill", &path.replace("\"", "")])
+                    .output()?;
```
Comment (vineetred) [inline src/wallpapers/linux.rs:117]: Introduces dependecy ```feh```. Look for alternate solution.

**Review by vineetred — CHANGES_REQUESTED** [review]

**Comment by joseildofilho** [comment]
@vineetred I made some research about it, and, i couldn't find another way to solve it.

**Comment by vineetred** [comment]
@joseildofilho have you tested this on a BSPWM machine?

**Comment by joseildofilho** [comment]
yep, i did it

**Comment by vineetred** [comment]
@joseildofilho I shall merge it then. There are some conflicts so can you rebase and push again?

**Pull request merged**